sql/core/src/test/scala/org/apache/spark/sql/execution/command/v2/ShowTablesSuite.scala

```diff
@@ -18,8 +18,9 @@
 package org.apache.spark.sql.execution.command.v2
 
 import org.apache.spark.sql.{AnalysisException, Row}
-import org.apache.spark.sql.catalyst.analysis.NoSuchNamespaceException
+import org.apache.spark.sql.catalyst.analysis.{NoSuchNamespaceException, NoSuchPartitionException}
 import org.apache.spark.sql.execution.command
+import org.apache.spark.util.Utils
 
 /**
  * The class contains tests for the `SHOW TABLES` command to check V2 table catalogs.
```

Comment:
is there any test case we can move to the base test suite now? Otherwise it's hard to track the behavior difference between v1 and v2 command.

Reply:
Done.